src/components/Pay2Win.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <div class="row">
+    <div>
+      <p class="h6 text-light bg-primary p-3 mt-3 mb-1 rounded">
+        Track your alchemy progress of pay2win upgrades and see the costs/bonuses of those upgrades.
+      </p>
+    </div>
+  </div>
+  <p class="text-light"> THIS WILL BE THE P2W PAGE {{gold}} </p>
+  <Pay2WinRow/>
+</template>
+
+<script lang="ts">
+  import Pay2WinRow from "~/components/Pay2WinRow.vue";
+
+  import { defineComponent } from "vue";
+  import { Alch } from "~/composables/Alchemy";
+  import { Util } from "~/composables/Utilities";
+
+  export default defineComponent({
+    name: "Pay2Win",
+    components: {
+      Pay2WinRow,
+    },
+    setup() {
+      let gold = Alch['Pay2Win'](50, 51, 0, 0, 0)[0];
```

Comment:
```suggestion
      let gold = Alch.Pay2Win(50, 51, 0)[0];
```

Adjust the argument count to the 3 `Alch.Pay2Win` actually takes.

Also the gold variable gets infered to type `string|number`. This is due to the fact that the first assigned value is of type number but in the next line gold is assigned to a value of type string. I suggest adding a temp variable here to prevent this type issue.

Reply:
The pay2win page is not a priority for this release, however I implemented these changes for a possible further release.